python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -648,7 +648,7 @@ def radius_servers(self) -> None:
                 )
 
             server_key = self._get_tacacs_or_radius_server_password(server)
```

Comment:
so this would raise is both key and cleartext_key are missing according to the function. But the schema is stating that you will ignore it. So we end up requiring something when actually it might not be needed right?

the only test you have configured is adding a key so we will not see this.

Maybe you should have something like `if not server.tls:` to guard sevrer_key but now we end up doing something uniquely because of the ignore in the schema which is meh...

Today we required key or cleartext key because we want traffic to be encrypted. if TLS removes this needs then we should change `_get_tacacs_or_radius_server_password` to not raise if tls is set.

@ClausHolbechArista for comment

Reply:
discussed - let's do an `if not server.tls:` ... 
```suggestion
            if not server.tls:
                server_key = self._get_tacacs_or_radius_server_password(server)
```

(server_key won't be set for 651 so need more adjustment)